apps/speed.c

```diff
@@ -1695,9 +1729,9 @@ int speed_main(int argc, char **argv)
             break;
         case OPT_EVP:
             evp_md = NULL;
```

Comment:
We should probably have a check somewhere to see if "-evp" has already been specified. Previously doing this was nonsensical, but harmless:
````
openssl speed -evp aes-128-cbc -evp aes-256-cbc
````
You can only specify "-evp" once, but if you do specify it twice the first instance is ignored. Now though, with the changes in this PR,  there will be a mem leak in this case.

Reply:
Done. Thanks!